build.gradle.kts

```diff
@@ -157,6 +159,34 @@ if (curse_api_key != null) {
         }
     }
 }
+val modrinth_api_key: String? = System.getenv("MODRINTH_TOKEN")
+if (modrinth_api_key != null) {
+    val MODRINTH_ID: String by project
+    modrinth {
+        projectId.set(MODRINTH_ID)
+        gameVersions.set(
+            listOf(
+                "1.14", "1.14.1", "1.14.2", "1.14.3", "1.14.4",
+                "1.15", "1.15.1", "1.15.2",
+                "1.16", "1.16.1", "1.16.2", "1.16.3", "1.16.4", "1.16.5",
+                "1.17", "1.17.1",
+                "1.18", "1.18.1", "1.18.2",
+                // "1.19",
+            )
+        )
```

Comment:
this should be abstracted to a global list and used for both the curseforge and the modrinth task